casper/src/main/scala/coop/rchain/casper/util/comm/CasperPacketHandler.scala

```diff
@@ -387,7 +387,7 @@ object CasperPacketHandler extends CasperPacketHandlerInstances {
         _     <- Log[F].info(s"Received ForkChoiceTipRequest from $peer")
         tip   <- MultiParentCasper.forkChoiceTip
         local <- RPConfAsk[F].reader(_.local)
-        msg   = Blob(local, Packet(transport.ForkChoiceTipRequest.id, tip.toByteString))
+        msg   = Blob(local, Packet(transport.BlockMessage.id, tip.toByteString))
```

Comment:
@rabbitonweb @sebtomba maybe we should think about an API that prevents such mishaps :)